internals/overlord/servstate/manager_test.go

```diff
@@ -351,8 +351,17 @@ services:
 
 	stops, starts, err := s.manager.Replan()
 	c.Assert(err, IsNil)
-	c.Check(stops, DeepEquals, []string{"test2"})
-	c.Check(starts, DeepEquals, []string{"test1", "test2"})
+	for _, row := range stops {
```

Comment:
Why can't we do a direct `DeepEquals` check, like `c.Check(stops, DeepEquals, [][]string{{"test2", "test1"}})`?

Reply:
Good spot! In my first draft, the first index of the lanes 2D array is the lane ID, and `lanes[0]` could be empty, so I ended up with this version. Later I changed it and it could not be empty any more. I will change it to a direct `DeepEquals`.